src/main/kotlin/dev/proxyfox/command/menu/CommandMenu.kt

```diff
@@ -0,0 +1,39 @@
+package dev.proxyfox.command.menu
+
+import dev.proxyfox.command.CommandContext
+import dev.proxyfox.command.ScreenBuilder
+
+public abstract class CommandMenu {
+    public var default: Pair<String, ScreenBuilder>? = null
+    public var active: CommandScreen? = null
+    private var screens: HashMap<String, ScreenBuilder> = hashMapOf()
+
+    public fun default(name: String = "default", action: ScreenBuilder) {
+        screens[name] = action
+        default = Pair(name, action)
```

Comment:
Maybe either or, not both. `ScreenBuilder?` would likely be preferable here given that it would not require traversing the hashmap.